src/serlio/prtMaterial/PRTMaterialNode.cpp

```diff
@@ -87,276 +94,148 @@ MStatus PRTMaterialNode::compute(const MPlug& plug, MDataBlock& block) {
-					mShadingCmd +=
-					        "$shadingNodeIndex = `shaderfx -sfxnode $shName -getNodeIDByName \"Standard_Base\"`;\n";
-					mShadingCmd += "shaderfx -sfxnode $shName -edit_stringlist $shadingNodeIndex blendmode " +
-					               blendMode + ";\n";
-
-					// ignored: ambientColor, specularColor
-					setAttribute(mShadingCmd, "diffuse_color", matInfo.diffuseColor);
-					setAttribute(mShadingCmd, "emissive_color", matInfo.emissiveColor);
-					setAttribute(mShadingCmd, "opacity", matInfo.opacity);
-					setAttribute(mShadingCmd, "roughness", matInfo.roughness);
-					setAttribute(mShadingCmd, "metallic", matInfo.metallic);
-
-					// ignored: specularmapTrafo, bumpmapTrafo, occlusionmapTrafo
-					// shaderfx does not support 5 values per input, that's why we split it up in tuv and swuv
-					setAttribute(mShadingCmd, "colormap_trafo_tuv", matInfo.colormapTrafo.tuv());
-					setAttribute(mShadingCmd, "dirtmap_trafo_tuv", matInfo.dirtmapTrafo.tuv());
-					setAttribute(mShadingCmd, "emissivemap_trafo_tuv", matInfo.emissivemapTrafo.tuv());
-					setAttribute(mShadingCmd, "metallicmap_trafo_tuv", matInfo.metallicmapTrafo.tuv());
-					setAttribute(mShadingCmd, "normalmap_trafo_tuv", matInfo.normalmapTrafo.tuv());
-					setAttribute(mShadingCmd, "opacitymap_trafo_tuv", matInfo.opacitymapTrafo.tuv());
-					setAttribute(mShadingCmd, "roughnessmap_trafo_tuv", matInfo.roughnessmapTrafo.tuv());
-
-					setAttribute(mShadingCmd, "colormap_trafo_suvw", matInfo.colormapTrafo.suvw());
-					setAttribute(mShadingCmd, "dirtmap_trafo_suvw", matInfo.dirtmapTrafo.suvw());
-					setAttribute(mShadingCmd, "emissivemap_trafo_suvw", matInfo.emissivemapTrafo.suvw());
-					setAttribute(mShadingCmd, "metallicmap_trafo_suvw", matInfo.metallicmapTrafo.suvw());
-					setAttribute(mShadingCmd, "normalmap_trafo_suvw", matInfo.normalmapTrafo.suvw());
-					setAttribute(mShadingCmd, "opacitymap_trafo_suvw", matInfo.opacitymapTrafo.suvw());
-					setAttribute(mShadingCmd, "roughnessmap_trafo_suvw", matInfo.roughnessmapTrafo.suvw());
-
-					// ignored: bumpMap, specularMap, occlusionmap
-					setTexture(mShadingCmd, "color_map", matInfo.colormap);
-					setTexture(mShadingCmd, "dirt_map", matInfo.dirtmap);
-					setTexture(mShadingCmd, "emissive_map", matInfo.emissiveMap);
-					setTexture(mShadingCmd, "metallic_map", matInfo.metallicMap);
-					setTexture(mShadingCmd, "normal_map", matInfo.normalMap);
-					setTexture(mShadingCmd, "roughness_map", matInfo.roughnessMap);
-					setTexture(mShadingCmd, "opacity_map", matInfo.opacityMap);
-
-					swprintf(buf.data(), buf.size() - 1, L"sets -forceElement $sgName %ls.f[%d:%d];\n",
-					         MString(meshName).asWChar(), faceStart, faceEnd);
-					mShadingCmd += buf.data();
-				}
-
-				MCHECK(MGlobal::executeCommandOnIdle(mShadingCmd, DBG));
-			}
-		}
-
-		outputMesh.setMetadata(outputAssociations);
-	}
-	outMeshHandle.setClean();

```

Comment:
as discussed offline:
* the missing setClean is a regression and must be added
* we omit setMetadata, the long-term goal is to avoid persisting this metadata alltogether (see #70)

Reply:
re-added the setClean